docs/processes.md

```diff
@@ -0,0 +1,127 @@
+created in IBS and that request goes through a review process and through the staging workflow. But
+the set of tools used during the review process is not identical. On one hand, some tools may be
+common but configured in a slightly different way. On the other hand, there are some extra tools
+like the [Origin Manager](./origin-manager.md) to verify aspects that are not relevant for
+Tumbleweed.
+
+The following [SUSE-internal
+document](https://confluence.suse.com/display/projectmanagement/Product+Handbook) offers all kind of
+details about the processes involved in the development of SLE and all its associated products.
+
+## openSUSE Leap Development
+
+Starting with version 15.3, openSUSE Leap shares its base binary packages with the corresponding
+version of SUSE Enterprise Linux (SLE-15-SP3 in the case of Leap 15.3). This development process
+based on binaries generated in IBS was already tested as a proof of concept for Leap 15.2 under the
+codename Jump.
+
+TODO: document better this process and its relationships with the tools in this repository. For the
+time being, check [this section](https://en.opensuse.org/openSUSE:Packaging_for_Leap) of the
+openSUSE wiki and also [this description](https://en.opensuse.org/openSUSE:Leap_development_process)
+of the old (prior to Leap 15.3) development process.
+
+## Maintenance Process for openSUSE Leap and SLE
+
+Having a defined process to release maintenance updates is crucial for distributions like SLE and
+openSUSE Leap, which release a new version approximately once per year instead of getting a constant
+stream of updated packages like it happens in openSUSE Tumbleweed. Those maintenance updates are the
+only mechanism to fix severe bugs and security issues between two releases. 
+
+As explained in the corresponding section, during the development of a product, packages are checked
+into its GA project (eg. `SUSE:SLE-15-SP2:GA` is the corresponding project for SUSE Linux Enterprise
+15 SP2). Once the distribution is considered as ready to be released, the GA project is locked and a
+corresponding update project (like `SUSE:SLE-15-SP2:Update`) is introduced to receive packages that
+will be released as updates.
+
+Maintenance requests (similar to the submit requests created during development) are created to
+release an update of a package for an already released distribution. Those requests also go through
+a review process assisted by some of the tools in this repository. Similar to the staging workflow,
+in which one or more packages are verified in a staging project before their submit requests can be
+accepted into the GA project, the packages from one or several maintenance request are verified in a
+so-called incident project before accepting them in the corresponding update project.
+
+Despite the similarities between the concept of maintenance incidents and the staging workflow, the
+former is implemented by separate mechanisms not based on any of the tools in this repository.
```

Comment:
Maybe we could link the [OBS maintenance support](https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.maintenance_setup.html) page here.

Reply:
We definitely should! I have overlooked that one. Good catch.